test/integration/shells_int_test.go

```diff
@@ -0,0 +1,99 @@
+package integration
+
+import (
+	"fmt"
+	"runtime"
+	"testing"
+
+	"github.com/ActiveState/cli/internal/testhelpers/e2e"
+	"github.com/ActiveState/cli/internal/testhelpers/tagsuite"
+	"github.com/stretchr/testify/suite"
+)
+
+type ShellsIntegrationTestSuite struct {
+	tagsuite.Suite
+}
+
+func (suite *ShellsIntegrationTestSuite) TestShells() {
+	suite.OnlyRunForTags(tagsuite.Critical, tagsuite.Shell)
+
+	ts := e2e.New(suite.T(), false)
+	defer ts.Close()
+
+	var shells []e2e.Shell
+	switch runtime.GOOS {
+	case "linux":
+		shells = []e2e.Shell{e2e.Bash, e2e.Fish}
+	case "darwin":
+		shells = []e2e.Shell{e2e.Bash, e2e.Zsh, e2e.Tcsh}
```

Comment:
Mac has Fish shell too

Reply:
But it's not installed. Are we supposed to install it on CI for this test?

Reply:
Yes. The AC's said which shells we want to cover with tests. CI not having a shell installed doesn't alter the AC's, but we can certainly talk if it creates a big scope creep.